packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -138,6 +157,66 @@ class Checkbox extends StatefulWidget {
 }
 
 class _CheckboxState extends State<Checkbox> with TickerProviderStateMixin {
+  bool get enabled => widget.onChanged != null;
+  Map<LocalKey, ActionFactory> _actionMap;
+  bool _showHighlight = false;
+
+  @override
+  void initState() {
+    super.initState();
+    _actionMap = <LocalKey, ActionFactory>{
+      SelectAction.key: _createAction,
+      if (!kIsWeb) ActivateAction.key: _createAction,
```

Comment:
Are we currently using `kIsWeb` as a marker for desktop?

Also, is a single line `if` clause the preferred style for this in these conditional data structures? I kinda like it, but it does seem different than the `if` statement counterpart.